package.json

```diff
@@ -27,21 +27,21 @@
     "strongloop"
   ],
   "dependencies": {
-    "async": "^0.9.0",
-    "chalk": "^0.5.0",
-    "debug": "^2.1.0",
-    "inflection": "^1.3.8",
+    "async": "^1.4.2",
+    "chalk": "^1.1.0",
+    "debug": "^2.2.0",
+    "inflection": "^1.7.1",
     "loopback-swagger": "^2.0.0",
-    "loopback-workspace": "^3.15.0",
-    "request": "^2.47.0",
-    "yeoman-generator": "^0.18.7",
-    "yosay": "^1.0.0"
+    "loopback-workspace": "^3.16.0",
+    "request": "^2.60.0",
+    "yeoman-generator": "^0.20.2",
+    "yosay": "^1.0.5"
   },
   "devDependencies": {
-    "chai": "^1.10.0",
-    "fs-extra": "^0.12.0",
-    "jshint": "^2.5.8",
-    "mocha": "^2.0.1",
+    "chai": "^3.2.0",
+    "fs-extra": "^0.23.1",
+    "jshint": "^2.8.0",
+    "mocha": "^2.2.5",
```

Comment:
The following dependency upgrades appear to be unnecessary because they won't actually change the version that gets installed:
- debug
- inflection
- loopback-workspace
- request
- yosay
- jshint
- mocha

Unless this PR includes changes that actually requires features added in the new minimum versions, all these upgrades do is make the diff bigger and add more to read later if someone is investigating a bug.